### ISSUE TITLE
svd-matrix-inverse probabbly should be exported

### ISSUE THREAD

**Issue by sirherrbatka** [gwkkwg/cl-mathstats#3]
I was looking for this function, and cl-mathstats has it, but it is not exported. Is it intentional?

**Comment by sirherrbatka** [comment]
Same applies to SVD in fact.

**Comment by gwkkwg** [comment]
I completely agree. Unfortunately, I not currently working in or on Lisp related projects. If you'd like to submit a push request, I'll happily apply it. I'd also be happy to turn the project over to you. Just let me know.

**Comment by sirherrbatka** [comment]
Ok, I've created pull requests. I am not a mathematician, I am unfit to take over the project sadly.
